docs/kratos/self-service/flows/_common/code-link-comparison.mdx

```diff
@@ -0,0 +1,21 @@
+### Comparison
```

Comment:
The problem with saving the heading in the imported snippet is that it's not included in the docusaurus ToC for the doc (right-hand nav). ;(

<img width="1044" alt="image" src="https://user-images.githubusercontent.com/29116492/206226879-1645a592-29f8-4d69-8405-1d5769d8fe3d.png">


My recommendation - remove the h3 from the snippet. Add the heading manually to each doc, then import the snippet. Then you have both - reusable stuff and a heading included in ToC. :)

Reply:
Argh, damn. done.